applications/FluidDynamicsApplication/custom_processes/distance_smoothing_process.cpp

```diff
@@ -122,7 +120,9 @@ void DistanceSmoothingProcess<TDim, TSparseSpace, TDenseSpace, TLinearSolver>::E
 
             auto& n_nodes = rNode.GetValue(NEIGHBOUR_NODES);
             for (unsigned int j = 0; j < n_nodes.size(); ++j) {
-                if (!(n_nodes[j].Is(CONTACT)) || !(rNode.Is(CONTACT))){
+                if ( (n_nodes[j].Is(CONTACT) && rNode.Is(CONTACT))
+                    || (!rNode.Is(CONTACT) && !n_nodes[j].Is(CONTACT)) ){
```

Comment:
```suggestion
                if (n_nodes[j].Is(CONTACT) == rNode.Is(CONTACT)){
```
Aren't your changes equivalent to this?

Reply:
Seems the same.